Dockerfiles/agent/README.md

```diff
@@ -4,25 +4,65 @@ This is how the official agent 6 image available [here](https://hub.docker.com/r
 
 ## How to run it
 
-The following environment variables are supported:
+Head over to [datadoghq.com](https://app.datadoghq.com/account/settings#agent/docker) to get the official installation guide.
+
+For a simple docker run, you can quickly get started with:
+
+```
+docker run -d -v /var/run/docker.sock:/var/run/docker.sock:ro \
+              -v /proc/:/host/proc/:ro \
+              -v /sys/fs/cgroup/:/host/sys/fs/cgroup:ro \
+              -e DD_API_KEY=<YOUR_API_KEY> \
+              datadog/agent:latest
+```
+
+The agent is highly customizable, here are the most used environment variables:
+
+#### Global options
 
 - `DD_API_KEY`: your API key (**required**)
-- `DD_HOSTNAME`: hostname to use for metrics
+- `DD_HOSTNAME`: hostname to use for metrics (if autodetection fails)
 - `DD_TAGS`: host tags, separated by spaces. For example: `simple-tag-0 tag-key-1:tag-value-1`
 
-- `DD_DOGSTATSD_NON_LOCAL_TRAFFIC`: listen to dogstatsd packets from other containers, required to send custom metrics
+#### Optional collection agents
+
 - `DD_APM_ENABLED`: run the trace-agent along with the infrastructure agent, allowing the container to accept traces on 8126/tcp
 - `DD_PROCESS_AGENT_ENABLED`: run the [process-agent](https://docs.datadoghq.com/graphing/infrastructure/process/) along with the infrastructure agent, feeding data to the Live Process View and Live Containers View
 - `DD_LOGS_ENABLED`: run the [log-agent](https://docs.datadoghq.com/logs/) along with the infrastructure agent. See below for details
-- `DD_JMX_CUSTOM_JARS`: space-separated list of custom jars to load in jmxfetch (only for the `-jmx` variants)
+
+#### Dogstatsd (custom metrics)
+
+You can send custom metrics via [the statsd protocol](https://docs.datadoghq.com/developers/dogstatsd/):
+
+- `DD_DOGSTATSD_NON_LOCAL_TRAFFIC`: listen to dogstatsd packets from other containers, required to send custom metrics
+
+#### Tagging
+
+We automatically collect common tags from [Docker](https://github.com/DataDog/datadog-agent/blob/master/pkg/tagger/collectors/docker_extract.go), [Kubernetes](https://github.com/DataDog/datadog-agent/blob/master/pkg/tagger/collectors/kubelet_extract.go), [ECS](https://github.com/DataDog/datadog-agent/blob/master/pkg/tagger/collectors/ecs_extract.go), [Swarm, Mesos, Nomad and Rancher](https://github.com/DataDog/datadog-agent/blob/master/pkg/tagger/collectors/docker_extract.go), and allow you to extract even more tags with the following options:
+
+- `DD_DOCKER_LABELS_AS_TAGS` : extract docker container labels
+- `DD_DOCKER_ENV_AS_TAGS` : extract docker container environment variables
+- `DD_KUBERNETES_POD_LABELS_AS_TAGS` : extract pod labels
+
+You can either define them in your custom `datadog.yaml`, or set them as JSON maps in these envvars. The map key is the source (label/envvar) name, and the map value the datadog tag name. If you prefix the tag name with +, it will only be added to high cardinality metrics (docker check). For example:
```

Comment:
given we're moving to handling high-card tags in the backend, I don't think we need to document the last part